libraries/AP_Scripting/drivers/mount-djirs2-driver.lua

```diff
@@ -727,7 +727,7 @@ function update()
     end
 
     -- send rate target
-    local roll_degs, pitch_degs, yaw_degs, yaw_is_ef = mount:get_rate_target(MOUNT_INSTANCE)
```

Comment:
aha!  this is a real bug because we should be using this.  I will create a fix although I'm happy for this to go in as well.